campaignion_language_switcher/campaignion-language-switcher.tpl.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @file
+ * Display a list of translation links.
+ *
+ * Available variables:
+ * - $links: An array of $link-arrays. Each $link in $links contains:
+ *   - $link['li_attributes']: Attribute array for the list-item.
+ *   - $link['renderable']: A renderable array for the link content.
+ *
+ * @see template_preprocess_campaignion_language_switcher()
+ *
+ * @ingroup themeable
+ */
+?>
+<ul class="campaignion-language-switcher-<?php echo $provider; ?>">
```

Comment:
Wrapping it in a `<nav class="campaignion-language-switcher">` would be better markup (after all, it has a menu role), but would break backward compatability with current themes, I guess...

Reply:
Yep, there a many themes with `ul.campaignion-language-switcher` selectors.